portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function addQuote() {
+    const quote =
```

Comment:
[4.2](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces.